### DISCUSSION TITLE
türkçe destek

### DISCUSSION THREAD

**Discussion by TaYRoN99** [IBAX-io/go-ibax#899] [Ideas]
### Clear and concise description of the problem

türkçe dil desteği lazım 

### Suggested solution

türkish language support 

### Alternative

türkçe olmalı 

### Additional context

türkçe olsun lütfen 

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by mamilincoln** [comment]
Kesinlikle olması lazım

**Comment by erayerdogantr** [comment]
Türkçe dil istiyoruz.
We want Turkish language .

**Comment by 2558Mete** [comment]
Çok iyi olur

**Comment by kriptodalgic67** [comment]
türkçe dill..

**Comment by Alikyuncu** [comment]
Türkçe dil desteği olursa çok daha güzel olur

**Comment by Bad19951453** [comment]
Türkçe dili olması daha verimli kılar en azından bizim için

**Comment by ttuna1** [comment]
türkçe gelsin

**Comment by ibaxio** [comment]
The IBAX team is recruiting volunteers to translate the official website. Due to the rapid growth of the Turkish community, Turkish will be uploaded in the next revision.

We also welcome translations from community members in other languages.

Unleash the power of our community and make us stronger.
